tests/Unit/CustomerTest.php

```diff
@@ -231,6 +232,46 @@ class CustomerTest extends TestCase
       "updated_at": "2015-06-09T13:16:00-04:00"
     }';
 
+    const LIST_OPPORTUNITIES_JSON= '{
+      "entries": [
+        {
+          "uuid": "00000000-0000-0000-0000-000000000000",
+          "customer_uuid": "cus_00000000-0000-0000-0000-000000000000",
+          "owner": "test1@example.org",
+          "pipeline": "New business 1",
+          "pipeline_stage": "Discovery",
+          "estimated_close_date": "2023-12-22",
+          "currency": "USD",
+          "amount_in_cents": 100,
+          "type": "recurring",
+          "forecast_category": "pipeline",
+          "win_likelihood": 3,
+          "custom": {"from_campaign": true},
```

Comment:
I think custom type is array in [here](https://github.com/chartmogul/chartmogul-php/pull/124/files#diff-ee0538044ba8a09f6c14c0df74cd102dea15b7983d71e9b0ff7751b18d2f39e7R25) but this type is object

Reply:
Yes, good catch, but the data should be correct now 😊 
I just added a new test for creating the opportunity from customer.